docs/src/modules/components/AppSearch.js

```diff
@@ -153,28 +154,33 @@ function NewStartScreen() {
   );
 }
 
-function DocSearchHit(props) {
-  const { children, hit } = props;
+const displayTagProductId = {
+  'material-ui': 'Material UI',
+  'joy-ui': 'Joy UI',
+  'base-ui': 'Base UI',
+  x: 'MUI X',
```

Comment:
Fix badge for MUI X

Before:
<img width="751" alt="Screenshot 2023-06-14 at 17 18 03" src="https://github.com/mui/material-ui/assets/3165635/7d28d7a1-9550-4e62-ade5-6e57b45dc630">

After:

<img width="747" alt="Screenshot 2023-06-14 at 17 18 43" src="https://github.com/mui/material-ui/assets/3165635/5e3e7bf6-6047-4ee4-ac88-1c7e7ac78806">